code/modules/jobs/job_mail.dm

```diff
@@ -77,6 +77,8 @@
 
 		//NSV13
 		/obj/item/reagent_containers/food/drinks/drinkingglass/doctor_delight = 8,
+		/obj/item/reagent_containers/hypospray/medipen/stimpack = 2,
```

Comment:
This item is restricted to ERT, Antags, and admin. It is a highly powerful healing item. 10 u is enough to heal most spessmen a massive amount

Reply:
I'm in the middle of removing all the powercreep items, so give it time.